### PR TITLE
Update apps to 212 release

### PR DESCRIPTION
**Pull request by rileymacdonald-sf**
Updated all of the example applications to the latest 212 release.

### CODE REVIEW

File: Examples/SnapinsSDKExample/app/src/main/java/com/salesforce/snapinssdkexample/SnapinsApplication.java [hunk 1]
```diff
@@ -9,40 +9,20 @@
...
-                // WARNING: WE HIGHLY RECOMMEND NOT HARDCODING THIS
-                // IN YOUR APP!!! THIS IS USED ONLY AS AN EXAMPLE!!!
-                // Refer to the Salesforce Mobile SDK for more info.
-                // https://developer.salesforce.com/docs/atlas.en-us.mobile_sdk.meta/mobile_sdk/offline_intro.htm
-                return Encryptor.hash(name +
-                        "12s9adpahk;n12-97sdainkasd=012", name +
-                        "12kl0dsakj4-cxh1qewkjasdol8");
-              }
-            }, SnapinsActivity.class);
+    SalesforceSDKManager.initNative(this, null, SnapinsActivity.class);
```
Comment (sfyz) [inline Examples/SnapinsSDKExample/app/src/main/java/com/salesforce/snapinssdkexample/SnapinsApplication.java]: Is this necessary for this example?

File: Examples/SnapinsSDKExample/app/build.gradle [hunk 2]
```diff
@@ -49,17 +49,19 @@ dependencies {
   * If you only use a single Snap-In just pull in the Snap-In you need.
   * compile "com.salesforce.service:case-ui:<VersionNumber>"
   */
-  compile 'com.salesforce.service:servicesdk:210.0.0'
+  implementation "com.salesforce.service:servicesdk:$serviceSdkVersion"
+  implementation "com.salesforce.mobilesdk:SalesforceSDK:$mobilesdkVersion"
```
Comment (sfyz) [inline Examples/SnapinsSDKExample/app/build.gradle]: Is this dependency necessary for this example?

File: Examples/SnapinsSDKExample/app/src/main/java/com/salesforce/snapinssdkexample/SnapinsApplication.java (hunk 1 above)
Comment (rileymacdonald-sf) [reply Examples/SnapinsSDKExample/app/src/main/java/com/salesforce/snapinssdkexample/SnapinsApplication.java]: No necessary here, I assumed this example was using authentication. I'll remove mobile SDK from this pull request. Good catch!

File: Examples/SnapinsSDKExample/app/build.gradle (hunk 2 above)
Comment (rileymacdonald-sf) [reply Examples/SnapinsSDKExample/app/build.gradle]: Removing per my comment above.

**Pull request merged**